src/index.ts

```diff
@@ -151,10 +149,17 @@ const mise = async (args: string[]): Promise<number> =>
       core.getInput('working_directory') ||
       core.getInput('install_dir') ||
       process.cwd()
+    const env = core.isDebug()
+      ? { ...process.env, MISE_LOG_LEVEL: 'debug' }
```

Comment:
```suggestion
      ? { MISE_LOG_LEVEL: 'debug', ...process.env }
```

if the log level is already set (say, to "trace") we should respect that

Reply:
Amended the fix to the commit.